README.md

```diff
@@ -15,6 +15,12 @@ pip install aioswitcher
   <td><a href="https://github.com/TomerFi/aioswitcher/blob/dev/CONTRIBUTING.md">Contributing</a></td>
 </table>
 
+<strong>
```

Comment:
This note might fit better in the [Wiki](https://github.com/TomerFi/aioswitcher/wiki).


Regardless, all-bold is just too much, IMO:

![image](https://github.com/TomerFi/aioswitcher/assets/28388442/9babb2b9-7a24-48f4-bd0c-b39241cc34c2)

Reply:
I am afraid it will be missed if it is in Wiki and not on the main screen.
If I remove the bold.. it will be enough?